myuw/static/vue/components/_common/course/student/course.vue

```diff
@@ -56,30 +56,26 @@
         <template
           v-if="section.is_ended || getSectionEval(section.index).length > 0"
         >
-          <b-button
-            v-b-toggle="`course-details-${index}`"
-            variant="link"
-            size="sm"
-            class="w-100 p-0 border-0 text-dark"
+          <button v-b-toggle="`course-details-${index}`"
+            type="button"
+            class="btn btn-link w-100 p-0 border-0 text-dark"
```

Comment:
`btn-sm` is missing here

Reply:
"No instructor information available" has myuw-text-md